### PR TITLE
3.37.0

### PR DESCRIPTION
**Pull request by greg0ire**
Just approve

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.37.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.36.0...3.37.0) - 2018-07-26
+### Fixed
+- fixed bug with complex properties in `ModelAutocompleteType`
+
+### Changed
+- Enable TimeZone for datetime and time templating
+- Changing the way of checking the permission
```
Comment (OskarStark) [inline CHANGELOG.md]: The permissions of what? Could be more precise here?

Comment (greg0ire) [reply CHANGELOG.md]: cc @axzx

**Comment by axzx** [comment]
Changing the way of checking the permissions to display a link in template. 

Could you correct a sentence above in english?;-)

File: CHANGELOG.md (hunk 1 above)
Comment (axzx) [reply CHANGELOG.md]: Changing the way of checking the permissions to display a link in template.

Could you correct a sentence above in english?;-)

**Pull request merged**